nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```
@@ -0,0 +1,27 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("./../data/reservations.json");
+
+router.get("/", async (request, response) => {
+    try {
+        console.log(reservations);
+        console.log("in /api/reservations");
+        response.send(reservations);
+    } catch (error) {
+        throw error;
```

throwing an error here is not helpful because it would crash your server app if it's not caught.
you can return an error code like `500 Internal Server Error` instead